<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="mcadmins-MBP-2.lan" ignored="0" name="NetBanking" tests="4" failures="2" timestamp="2023-06-15T12:16:30 EDT" time="23.259" errors="0">
  <testcase name="logintest" time="3.157" classname="com.qa.testCases.LoginTest"/>
  <testcase name="logintest" time="2.166" classname="com.qa.testCases.LoginTest_DataDriven"/>
  <testcase name="logintest" time="1.171" classname="com.qa.testCases.LoginTest_DataDriven">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.testCases.LoginTest_DataDriven.logintest(LoginTest_DataDriven.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
  <testcase name="logintest" time="1.13" classname="com.qa.testCases.LoginTest_DataDriven">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.testCases.LoginTest_DataDriven.logintest(LoginTest_DataDriven.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
</testsuite> <!-- NetBanking -->
